### PR TITLE
[406] Change allocation table to use card-list

### PR DESCRIPTION
**Pull request by stellenberger**
### Context

- [This trello ticket](https://trello.com/c/1dIdEqIM/406-change-table-type-in-the-support-console-allocations-to-horizontal)

### Changes proposed in this pull request

- Flip table horizontally to use a carded list instead

### Guidance to review

- Log onto support console and go to the PE Allocations tab

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by stellenberger** [comment]
[Change table type in the support console allocations to horizontal](https://trello.com/c/1dIdEqIM/406-change-table-type-in-the-support-console-allocations-to-horizontal)

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ Lint/MissingSuper:
     - "app/components/paginator/view.rb"
     - "app/components/providers/filters/view.rb"
     - "app/components/tab_navigation/view.rb"
+    - "app/components/allocation_record_card/view.rb"
```
Comment (defong) [inline .rubocop_todo.yml:56]: If this is here is it inheriting of the wrong parent 

Maybe use `ViewComponent::Base` or it really needs `super`

**Review by defong — CHANGES_REQUESTED** [review]

File: spec/components/allocation_record_card/view_preview.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+module AllocationRecordCard
+  class ViewPreview < ViewComponent::Preview
+    def single_card
+      render(AllocationRecordCard::View.new(allocation: mock_allocation))
+    end
+
+    def multiple_cards
+      render(AllocationRecordCard::View.with_collection(mock_multiple_allocations))
+    end
```
Comment (defong) [inline spec/components/allocation_record_card/view_preview.rb:11]: https://teacher-training-api-pr-2292.london.cloudapps.digital/view_components/allocation_record_card/view/multiple_cards

File: spec/components/allocation_record_card/view_preview.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AllocationRecordCard
+  class ViewPreview < ViewComponent::Preview
+    def single_card
+      render(AllocationRecordCard::View.new(allocation: mock_allocation))
+    end
```
Comment (defong) [inline spec/components/allocation_record_card/view_preview.rb:7]: https://teacher-training-api-pr-2292.london.cloudapps.digital/view_components/allocation_record_card/view/single_card

**Review by defong — CHANGES_REQUESTED** [review]
view preview does not work

**Review by mbocc — APPROVED** [review]
Amazing des skills - all looks good, all approved!

**Review by defong — APPROVED** [review]

**Pull request merged**